automation_tools/__init__.py

```diff
@@ -1314,7 +1314,7 @@ def product_install(distribution, create_vm=False, certificate_url=None,
             host=host
         )
     execute(fix_qdrouterd_listen_to_ipv6, host=host)
-    execute(enable_gateway_ports_connections, host=host)
```

Comment:
Since the `enable_gateway_ports_connections` will not be used anymore, please remove that task.